### PR TITLE
R0ad veh

### PR DESCRIPTION
**Pull request by r0adra93**
## Description

Beginning of solving an issue of category headings not displaying the way the server owners want them displayed.
Added to the table in shared/vehicles.lua.

## Checklist

- [x ] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [x ] My code fits the style guidelines.
- [x ] My PR fits the contribution guidelines.

### CODE REVIEW

**Review by jeann2013 — APPROVED** [review]

**Comment by Irishstevie** [comment]
bitmoji changes Are Good and ready to go :)

**Review by Irishstevie — APPROVED** [review]
changes Are Good and ready to go :)

**Comment by tom-osborne** [comment]
@r0adra93 Could you please look into the linting errors?

File: shared/vehicles.lua [hunk 1]
```diff
@@ -9,15 +9,17 @@ QBShared.Vehicles = {
 		['model'] = 'asbo',
 		['price'] = 4000,
 		['category'] = 'compacts',
+        ['categoryLabel'] = 'Compacts',
 		['hash'] = `asbo`,
-		['shop'] = 'pdm',
+		['shop'] = {'pdm','luxury'},
```
Comment (tom-osborne) [inline shared/vehicles.lua:14]: Any reason why you added `asbo` to the "luxury" vehicle shop?
Was this leftover from testing?

**Review by tom-osborne — CHANGES_REQUESTED** [review]
I appreciate this is not your fault, but would you mind also fixing the indentation throughout this file?

Replace any tab indents to 4 spaces. 

Example of currently inconsistent indentation:
image

File: shared/vehicles.lua (hunk 1 above)
Comment (r0adra93) [reply shared/vehicles.lua:14]: Yeah it was for testing to make sure multi stores worked.

Comment (r0adra93) [reply shared/vehicles.lua]: I removed the "luxury"

**Comment by r0adra93** [comment]
I replaced /t with /s/s/s/s that should solve the issue. I love regular expressions!

**Review by tom-osborne — APPROVED** [review]

**Pull request merged**